src/index.html

```diff
@@ -27,7 +27,7 @@
     <header class="page__section header">
       <div class="container">
         <div class="header__top">
-          <a href="/" class="header__logo">
+          <a href="#" class="header__logo">
```

Comment:
if you'll click logo in menu, you'll have same problem 404, href need to correct link, its can be header id